docs/organizations/segments.md

```diff
@@ -0,0 +1,37 @@
+# Segments
+Segments are dimensions that Codacy reads from your provider that organizes repositories into relevant groups for you. Today, Segments is available for: [GitHub Custom Properties](#github-custom-properties)
+  
+<br><br>
```

Comment:
Do we need all of these line breaks? This looks a bit odd to me. At first glance it seemed like an image was not loading or something like that.

![Screenshot 2024-10-09 at 11 26 15](https://github.com/user-attachments/assets/3017d21b-7911-4d11-902b-13d81a022142)